api/DynamicsCompressorNode.json

```diff
@@ -240,14 +232,8 @@
               "version_added": "6"
             },
             "safari_ios": "mirror",
-            "samsunginternet_android": {
-              "version_added": "1.0",
-              "notes": "Before Samsung Internet 6.0, this was an `AudioParam.`."
-            },
-            "webview_android": {
-              "version_added": "≤37",
-              "notes": "Before version 52, this was an `AudioParam.`."
-            },
+            "samsunginternet_android": "mirror",
```

Comment:
Mirroring changes version from 1.0 to 1.5, presumably because 1.0 is based on a WebKit version (between Safari 5.1 and Safari 6), whereas 1.5 is based on Blink.